core/encoding/xml/example/xml_example.odin

```diff
@@ -0,0 +1,55 @@
+package xml_example
+
+import "core:encoding/xml"
+import "core:mem"
+import "core:fmt"
+
+Error_Handler :: proc(pos: xml.Pos, fmt: string, args: ..any) {
+
+}
+
+FILENAME :: "../../../../tests/core/assets/xml/nl_NL-xliff-1.0.xliff"
```

Comment:
Wouldn't `"core:tests/tests/core/assets/xml/nl_NL-xliff-1.0.xliff"` work fine here?

Reply:
I don't know. Does `os.open` resolve those?

Reply:
`#load` does

Reply:
I just tried, but `tests` isn't a collection like `core` and `vendor` are, and `tests` doesn't live under core.

Reply:
You are correct, I am wrong.